roles/networking_mapper/molecule/default/converge.yml

```diff
@@ -214,3 +239,44 @@
                 _inventory_host_items | map(attribute='key') | list | sort
               )
             "
+
+    - name: Generate values for future deploy
+      ansible.builtin.import_role:
+        name: networking_mapper
+        tasks_from: network_config_values.yml
+
+    - name: Load generated file
+      register: _gen_snippet
+      ansible.builtin.slurp:
+        path: >-
+          {{
+            (ansible_user_dir, 'ci-framework-data',
+            'artifacts', 'ci_k8s_snippets',
+            'networking_mapper.yaml') | path_join
+          }}
+
+    - name: Ensure content is consistent
+      vars:
+        _content: "{{ _gen_snippet.content | b64decode | from_yaml }}"
+        _ctlplane_subnets:
+          - allocationRanges:
+              - end: 192.168.122.39
+                start: 192.168.122.30
+              - end: 192.168.122.59
+                start: 192.168.122.50
+            cidr: 192.168.122.0/24
+            gateway: 192.168.122.1
+            name: multus
+          - allocationRanges:
+              - end: 192.168.122.49
+                start: 192.168.122.40
+            cidr: 192.168.122.0/24
+            gateway: 192.168.122.1
+            name: netconfig
+      ansible.builtin.assert:
+        that:
```

Comment:
Isn´t this thing too loose? Should we use a golden file?

Reply:
Me neither, but, if we do not add tests here, in molecule, we can merge a broken jinja without notice. We will discover it's broken in the periodic e2e, far far later and probably with an error that doesn´t point directly to the real issue.

Reply:
I went for a "partial, split golden file" - seems easier to maintain on the medium term. Also took the main "test config" out in its own file, to make the `converge.yml` a bit smaller and easier to read.